Assets/Scenes/Help Scene.unity

```diff
@@ -38,7 +38,7 @@ RenderSettings:
   m_ReflectionIntensity: 1
```

Comment:
I think this is going to conflict with Alex's PR. Do you need to include game scene?